js/lbank.js

```diff
@@ -501,8 +501,22 @@ module.exports = class lbank extends Exchange {
             let query = this.keysort (this.extend ({
                 'api_key': this.apiKey,
             }, params));
-            let queryString = this.rawencode (query) + '&secret_key=' + this.secret;
-            query['sign'] = this.hash (this.encode (queryString)).toUpperCase ();
+            let queryString = this.rawencode (query);
+            let message = this.hash (this.encode (queryString)).toUpperCase ();
+            let secretArr = ['-----BEGIN RSA PRIVATE KEY-----'];
+            let secretLength = true ? this.secret.length : null;  // eslint-disable-line
+            for (let i = 0; i < secretLength; i++) {
+                let start = i * 64;
+                let end = start + 64;
```

Comment:
This won't work, though: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings

It has to be: `let end = this.sum (start, 64);`